opm/autodiff/StandardWellsDense_impl.hpp

```diff
@@ -187,7 +187,7 @@ namespace Opm {
         const double volume = 0.002831684659200; // 0.1 cu ft;
         for (int w = 0; w < nw; ++w) {
             bool allow_cf = allow_cross_flow(w, ebosSimulator);
-            const EvalWell bhp = getBhp(w);
+            const EvalWell& bhp = getBhp(w);
```

Comment:
I think non-trivial objects such as Evaluations should always be captured as const references if possible: this way it always works and it is always as performant as possible regardless of what the function likes to return.